src/csharp/Tokenizer.cs

```diff
@@ -65,13 +65,7 @@ public Sequences Encode(string str)
         public string Decode(ReadOnlySpan<int> sequence)
         {
             IntPtr outStr = IntPtr.Zero;
-            unsafe
-            {
-                fixed (int* sequencePtr = sequence)
-                {
-                    Result.VerifySuccess(NativeMethods.OgaTokenizerDecode(_tokenizerHandle, sequencePtr, (UIntPtr)sequence.Length, out outStr));
-                }
-            }
+            Result.VerifySuccess(NativeMethods.OgaTokenizerDecode(_tokenizerHandle, sequence.ToArray(), (UIntPtr)sequence.Length, out outStr));
```

Comment:
Same comment as setinputids. For spans, let's not change the implementation.

Reply:
👍Reverted